tests/Sniffs/NamingConventions/data/ValidVariableNameSniffTest.inc

```diff
@@ -0,0 +1,152 @@
+<?php
+$varName  = 'hello';
+$var_name = 'hello';
+$varname  = 'hello';
+$_varName = 'hello';
+
+class MyClass
+{
+$varName  = 'hello';
```

Comment:
tohle afaik není validní

naopak ti tu chybí:
- `var` (místo `public`/`protected`/`private`)
- varianta kde nespecifikuješ default value.
- ` = new something();`
- ` = Hello::world();`

Reply:
> tohle afaik není validní

co presne?

```
= new something();
= Hello::world();
```
co s timto? taky nechapu asi

Reply:
Přesně to u čeho komentuju.

![image](https://user-images.githubusercontent.com/972493/152360839-26c81ee3-3f4f-4c35-9e99-f50c9e112218.png)


ad ty dvě další věci: phpcs je hooooooodně specifický a sakra se vyplatí v testech mít "všechny možnosti, co existují", protože jinak tě to dřív nebo později skoro jistě kousne. Napsal jsem už desítky sniffů takže trust me, nechceš mít "částečný test", když jde o phpcs.

Reply:
welcome to the hell of writing phpcs sniffs no :D zkus kouknout na slevomat tooling, mají sakra moc užitečných funkcí, dost možná už na ty testy vymysleli něco lepšího, aby se nemuseli trackovat ty čísla řádků...